docusaurus/docs/dev-docs/configurations/features.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Features configuration
+sidebar_label: Features
+description: Enable experimental Strapi features
+displayed_sidebar: devDocsConfigSidebar
+---
+
+# Features configuration
+
+The `config/features.js|ts` file is used to enable feature flags. Currently this file only includes a `future` object used to enable experimental features through **future flags**. 
```

Comment:
```suggestion
The `config/features.js|ts` file is used to enable feature flags. Currently this file only includes a `future` object used to enable experimental features through **future flags**.  We do this so Developers have the flexibility to choose and integrate new features & changes into their Strapi applications as they become available in the current major version as well as assist us in shaping these new features.
```

Reply:
I included it, but not exactly there, just a bit higher above in the introduction :) Thank you!